libs/amp-evaluation/samples/langgraph-monitoring/.env.example

```
@@ -0,0 +1,12 @@
+# =============================================================================
+# OpenAI Configuration (for LLM-as-judge evaluators)
+# =============================================================================
+OPENAI_API_KEY=sk-your-openai-api-key-here
+
+# =============================================================================
+# AMP Platform Configuration (for fetching traces)
+# =============================================================================
+AMP_AGENT_UID=87b6ae50-bb7e-43ea-a11b-83391ee49667
+AMP_ENVIRONMENT_UID=80d9d1e4-5514-424a-8f0f-f00024aacfa7
+AMP_API_URL=http://localhost:9098
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The AMP_API_URL key should go before the AMP_ENVIRONMENT_UID key

(UnorderedKey)